apollo-integration/src/test/java/com/apollographql/apollo/internal/batch/BatchPollerTest.kt

```diff
@@ -0,0 +1,154 @@
+package com.apollographql.apollo.internal.batch
+
+import com.apollographql.apollo.Utils
+import com.apollographql.apollo.api.Input
+import com.apollographql.apollo.api.internal.ApolloLogger
+import com.apollographql.apollo.exception.ApolloException
+import com.apollographql.apollo.integration.interceptor.AllFilmsQuery
+import com.apollographql.apollo.interceptor.ApolloInterceptor
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.mock
+import java.util.concurrent.TimeUnit
+
+class BatchPollerTest {
```

Comment:
NIce tests 👍

Reply:
next I want to add a test with some fixtures to test the serialization / deserialization